apps/ubuntu_bootstrap/lib/pages/storage/storage_model.dart

```diff
@@ -187,14 +244,15 @@ class StorageModel extends SafeChangeNotifier {
       return 'resize_use_free';
     } else if (type == StorageType.manual) {
       return 'manual';
+    } else if (type case StorageTypeEraseInstall()) {
+      return 'reuse_partition';
```

Comment:
`reuse_partition` is probably not the right label here. The TODO note refers to an installation scenario we don't support (re-using an existing ubuntu installation and keeping the user's `/home` directory).
The label is only being used for the telemetry that is written to `/var/log/installer/telemetry`